src/pages/index.js

```diff
@@ -0,0 +1,7 @@
+export { default as Home } from "./Home/Home";
+export { default as Cart } from "./Cart/Cart";
+export { default as Wishlist } from "./Wishlist/Wishlist";
+export { default as Products } from "./Products/Products";
+export { default as Login } from "./Login/Login";
+export { default as Signup } from "./Signup/Signup";
+export { default as ProductPage } from "./ProductPage/ProductPage";
```

Comment:
This is something new I learnt where you export all the components from an index.js file

Reply:
Yes, it's called Re-exports. It allows you to do cleaner imports when working with many components of the same folder. You can read about it here [Javascript Re-export](https://javascript.info/import-export#re-export)